samcli/commands/build/command.py

```diff
@@ -126,6 +126,13 @@
     "requests=1.x and the latest request module version changes from 1.1 to 1.2, "
     "SAM will not pull the latest version until you run a non-cached build.",
 )
+@click.option(
+    "--container-env-vars",
```

Comment:
this option name feels a bit off. When I see it, I expect to pass the vars in the format the same as parameter overrides instead of a file.

Reply:
update: existing similar option https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-using-invoke.html#serverless-sam-cli-using-invoke-environment-file